lib/cn/cn.c

```diff
@@ -1001,8 +1001,8 @@ cn_open(
     /* If there's only one node then increase the split size to allow small trees
```

Comment:
This comment seems incorrect now.

Reply:
No, it will be extremely useful with zspill, a stripped down version of which should be easy to implement.

Reply:
Sure, but you changed the `if` condition, so you no longer check if there is only one node.

Reply:
Right, that's to keep the improvement if you close/reopen.  Say you run a time series test the close.  The re-open and resume, we want to retain the capability to leverage zspill for monotonically increasing loads.  We should probably also do the for monotonically decreasing loads, but I've never seen a test that does that (it's a bit unnatural...)

Reply:
Doh!  Will fix...

Reply:
I'll get it in next PR to avoid churn...